cli/src/main.rs

```diff
@@ -16,6 +16,16 @@ fn get_dockerfile_b64() -> String {
   return base64::encode(dockerfile);
 }
 
+fn get_env_file_b64(env_file_path: String) -> String {
+  let pwd = std::env::var("PWD").unwrap();
```

Comment:
nit: <https://doc.rust-lang.org/std/env/fn.current_dir.html> will work on all platforms Rust supports (that should also probably be used in `get_dockerfile_b64`)